README.md

```diff
@@ -36,103 +36,122 @@ export PATH="$PATH:$GEM_HOME/bin"
+```
+$ gem install bundler
+```
 
 ##### Set up Homebrew Version of Ruby
 
-If you're using `csh` or `tcsh` add the following lines to your `.cshrc` or `.tcshrc`,
+Because macOS provides its own version of Ruby, Homebrew doesn't automatically set up symlinks to access the version you justinstalled version with the `ruby` command. But after a successful install, Homebrew outputs the commands you'll need to run to set up the symlink yourself. If you use the default macOS `zsh` shell on Apple Silicon, you can set up the symlink with the following command:
 
 ```
-setenv PATH /usr/local/bin:/usr/local/sbin:$PATH
-
-setenv PATH /usr/local/opt/ruby/bin:${PATH}
-setenv PATH ${PATH}:/usr/local/lib/ruby/gems/2.7.0/bin
-setenv LDFLAGS -L/usr/local/opt/ruby@2.7/lib
-setenv CPPFLAGS -I/usr/local/opt/ruby@2.7/include
-setenv PKG_CONFIG_PATH /usr/local/opt/ruby@2.7/lib/pkgconfig
+$ echo 'export PATH="/opt/homebrew/opt/ruby/bin:$PATH"' >> ~/.zshrc
 ```
 
-or if you're using `bash` add the following lines to your `.bash_profile`,
+If you run macOS on an Intel-based Mac, replace `opt/homebrew` with `usr/local` in the above command.
+
+If you run a shell other than the default, check which config file to modify for the command. For instance, `bash` uses `~/.bashrc` or `~/.bash_profile`.
 
+Once you've made the changes to your shell configuration, open a new terminal instance and run the following command:
+
+```
+$ ruby --version
 ```
-export PATH="/usr/local/bin:/usr/local/sbin:$PATH"
 
-export PATH="/usr/local/opt/ruby/bin:$PATH"
-export PATH="$PATH:/usr/local/lib/ruby/gems/2.7.0/bin"
-export PATH="/usr/local/opt/ruby@2.7/bin:$PATH"
-export LDFLAGS="-L/usr/local/opt/ruby@2.7/lib"
-export CPPFLAGS="-I/usr/local/opt/ruby@2.7/include"
-export PKG_CONFIG_PATH="/usr/local/opt/ruby@2.7/lib/pkgconfig"
+You should see output similar to the following:
+
+```
+ruby 3.2.2 (2023-03-30 revision e51014f9c0) [arm64-darwin22]
 ```
-NOTE: If you are running macOS on an Apple Silicon based Mac, rather than an Intel Mac, substitute `/opt/homebrew/` for `/usr/local` in the lines dealing with `ruby@2.7` in the above block.
 
-and then open a new Terminal window to make sure you're using the right version of Python and Ruby.
+As long as you see a Ruby version greater than or equal to 3.2.2, you've succeeded.
 
-##### Install Dependencies
+##### Install Homebrew Dependencies
 
-Go ahead and `brew install` the other dependencies,
+Next, use Homebrew to install the other dependencies.
+Start with the latest version of Python:
 
 ```
 $ brew install python@3
+```
+
+Then install the [Ninja build system](https://formulae.brew.sh/formula/ninja#default):
+
+```
 $ brew install ninja
+```
+
+Then install the [Gumbo HTML5 parser](https://formulae.brew.sh/formula/gumbo-parser#default):
+
+```
 $ brew install gumbo-parser
+```
+
+And finally, install the [YAML module for Python 3](https://formulae.brew.sh/formula/pyyaml#default):
+
+```
 $ pip3 install pyyaml
 ```
```

Comment:
I _think_ this doesn't need to be here, because `pyyaml` gets installed anyway by the
```bash
pip3 install --user -r requirements.txt
```
command that we run later? (see https://github.com/raspberrypi/documentation/blob/develop/requirements.txt )

Reply:
If it works the way it's written, then it's fine.